packages/builder/yarn.lock

```diff
@@ -842,10 +842,17 @@
     lodash "^4.17.19"
     to-fast-properties "^2.0.0"
 
+<<<<<<< HEAD
```

Comment:
Little merge conflict here

Reply:
I knew it! I actually checked for this and Webstorm was like nah there's no errors in this file. Thanks for spotting.